xtask/src/main.rs

```diff
@@ -281,12 +350,23 @@ fn create_test_data() -> Result<()> {
     let disk = DiskParams {
         path: path.to_owned(),
         size_in_kilobytes: 1024 * 64,
+        fs_type: FsType::Ext4,
     };
     disk.create()?;
     disk.fill()?;
     disk.check()?;
     zstd_compress(&disk.path)?;
 
+    let path = dir.join("test_disk_ext2.bin");
+    let disk = DiskParams {
+        path: path.to_owned(),
+        size_in_kilobytes: 1024 * 96,
```

Comment:
Where's 96 coming from? Just "a number larger than that big math problem above"?

Reply:
Yeah 64 isn't quite big enough for the 3-level block map. Added a comment.